examples/basic_i3c_example.py

```diff
@@ -34,17 +27,15 @@ def main():
         print("I couldn't initialize the bus. Are you sure there's any target connected?")
         exit(1)
 
-    (_, targets) = i3c.targets()
-
     # Target PID in hexadecimal format
     target_pid = [0x00, 0x00, 0x00, 0x00, 0x6A, 0x04]
```

Comment:
This `target_pid` is reversed. The correct pid for the ICM42605 is [04-6A-00-00-00-00].